$NetBSD$

For some reason the previously used table_noop_array has a link error.

HACK: Create new array of noop functions.
BUG: Using environment variable LIBGL_DEBUG=verbose causes
massive failure in keyboard input for a Radeon 6450 (CAICOS) card.

--- src/mapi/entry.c.orig	2016-11-10 22:05:17.000000000 +0000
+++ src/mapi/entry.c
@@ -41,6 +41,1609 @@
 #define ENTRY_CURRENT_TABLE_GET U_STRINGIFY(u_current_get_table_internal)
 #endif
 
+#if defined(__NetBSD__) && defined(GLX_USE_TLS)
+static int netbsd_noop_generic(void) {
+    return 0;
+}
+
+/*
+ * src/mapi/table.h defines
+ *    MAPI_TABLE_NUM_SLOTS to be 
+ *      MAPI_TABLE_NUM_STATIC + MAPI_TABLE_NUM_DYNAMIC
+ *       = 1330 + 256 = 1586 entries
+ * according to values from mapi/shared-glapi/glapi_mapi_tmp.h
+ */
+
+const mapi_func netbsd_table_noop_array[] = {
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic,
+    (mapi_func)netbsd_noop_generic
+};
+#endif /* defined(__NetBSD__) && defined(GLX_USE_TLS) */
+
 #if defined(USE_X86_ASM) && defined(__GNUC__)
 #   ifdef GLX_USE_TLS
 #      include "entry_x86_tls.h"
